proseco/core.py

```diff
@@ -1450,3 +1450,34 @@ def get_kwargs_from_starcheck_text(obs_text, include_cat=False, force_catalog=Fa
         pass
 
     return kw
+
+
+def includes_for_obsid(obsid):
+    """
+    Return a dict with the include_ids_acq, include_halfws_acq and include_ids_guide
+    lists for the as-run flight catalog of ``obsid``.  Useful for forcing catalog.
```

Comment:
I note that get_starcheck_catalog has both as-run catalogs and has ingested latest-version catalogs for obsids that haven't actually run yet.  But that does not apply to this PR.